modules/reference/pages/cluster-properties.adoc

```diff
@@ -162,6 +162,34 @@ NOTE: The container must belong to <<cloud_storage_azure_storage_account,cloud_s
 
 ---
 
+=== cloud_storage_azure_hierarchical_namespace_enabled
+
+Whether or not Azure hierarchical namespaces (HNS) is enabled on <<cloud_storage_azure_storage_account>>. If this property is not set, then <<cloud_storage_azure_shared_key>> must be set, and each broker tries to determine at startup if HNS is enabled. Setting this property to `true` disables the check and assumes HNS is active. Setting this to `false` disables the check and assumes HNS is not active.
+
+*Type*: boolean
+
+*Default*: null
+
+*Restart required*: yes
+
+*Supported versions*: Redpanda v24.1 or later
+
+---
+
```

Comment:
this property is a `tunable` (not user-visible), it's not intended to be used during normal operations. 
I'm not sure what's the guidance for documenting these properties. maybe it needs a note like "users are not required to set this property"

Reply:
We are going to be creating an Internal Use Only section (or something similarly named) so that the properties that Support or Admins need get documented too.
@Deflaimun Please advise wrt where this is going to be placed. Thx.

Reply:
> this property is a `tunable` (not user-visible), it's not intended to be used during normal operations. I'm not sure what's the guidance for documenting these properties. maybe it needs a note like "users are not required to set this property"

Per docs guidance we'll remove the property from this page for now, until we've established where and how we document "internal only" properties.